[CN-1303]: add env var docs (#209)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/env-vars.adoc

= Configuring Environment Variables

You can configure the environment variables in the `hazelcast` pods using the `env` field of the CRD spec.

Example Configuration:
[source,yaml,subs="attributes+"]
----
include::ROOT:example$/env-vars.yaml[]
----

> WARNING: Environment variables to configure Hazelcast (environment variables with `HZ_` prefix) cannot be configured using `env` field. You should use xref:custom-config.adoc[Custom Config] to configure Hazelcast.

Reviewer: ```suggestion
WARNING: You cannot use the `env` field to configure Hazelcast environment variables, which are prefixed `HZ_`. To configure the Hazelcast environment variables, use xref:custom-config.adoc[Custom Config].
```
